packages/generate-EN10168-pdf-template/src/lib/createProductDescription.ts

```diff
@@ -0,0 +1,61 @@
+import { supplementaryInformation } from './supplementaryInformation';
+import { ProductDescription, Translate } from '../types';
+import { productNorms } from './productNorms';
+import { productShape} from './productShape';
+import {measurement} from './measurement';
+
+export function createProductDescription(productDescription: ProductDescription, i18n: Translate) {
+
+  const B02ProductNorms = productNorms(productDescription.B02, i18n);
+
+  const contentToOmit = ['B01', 'B02', 'B09', 'B10', 'B11', 'B12', 'B12','B13', 'SupplementaryInformation'];
+  const content = Object.keys(productDescription).filter(element => !contentToOmit.includes(element)).map(element =>
```

Comment:
That's almost identical to how we generate `content` in `commercialTransaction`. Perhaps worth extracting a function.